paddle/fluid/prim/utils/static/static_global_utils.h

```diff
@@ -93,7 +93,7 @@ class StaticCompositeContext {
   StaticCompositeContext()
       : current_block_desc_(nullptr),
         generator_(new UniqueNameGenerator()),
-        skip_comp_ops_({"matmul_v2"}) {}
+        skip_comp_ops_({"matmul_v2", "matmul_v2_grad"}) {}
```

Comment:
leave todo here to remove this when we fix static

Reply:
done